### PR TITLE
Issue/6006 split files

### LINKED ISSUE #6006: separate actions and helpers file into smaller files(and it's tests)
https://github.com/inmanta/web-console/pull/5985#discussion_r1812375670

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

Separate actions and helpers function into smaller files to improve readability of the code

closes #6006 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: src/UI/Components/Diagram/helpers/connetions.test.ts
Comment (LukasStordeur) [inline src/UI/Components/Diagram/helpers/connetions.test.ts:1]: Typo in file name you wrote connetions instead of connections.

**Review by LukasStordeur — COMMENTED** [review]
You're not using index files, but those would make the imports cleaner. You'll be able to import from `./helpers` or `@/helpers`, or `./actions` instead of needing to import each function separately when they happen to be spilt in a different file. 

I don't find `misc` a clear definition for the functions that are gathered there. Perhaps they could have been split up further? The test file is already considered too big to render the diff in the PR viewer, which indicates it might still be necessary to consider splitting it.

File: src/UI/Components/Diagram/helpers/connetions.test.ts
Comment (matborowczyk) [reply src/UI/Components/Diagram/helpers/connetions.test.ts:1]: fixed typo, added index files and renamed the misc file to "data" after a consideration it fit the description, they kind of related, and test files has 576 lines of code, which is still much but it's due to the fact that they manipulate objects. I think there is a chance that on the team  week we could erase some of them(helpers) if we could successfuly move to InstanceForm type of state management

File: src/UI/Components/Diagram/components/RightSidebar.test.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import {
   defaultCanvasContext,
 } from "../Context";
 import { containerModel } from "../Mocks";
-import { addDefaultEntities } from "../actions";
+import { addDefaultEntities } from "../actions/createMode";
```
Comment (LukasStordeur) [inline src/UI/Components/Diagram/components/RightSidebar.test.tsx:23]: update import

**Review by LukasStordeur — APPROVED** [review]

**Pull request closed without merging**